src/app/state/rest/hooks/useApiRequest.ts

```diff
@@ -10,7 +10,7 @@ type Config = {
   getHeaders?: () => Record<string, string>
 }
 
-type Callback = () => void
+export type Callback = () => void
```

Comment:
This can't be inferred? Exporting is only for unit tests?

Reply:
Yeah. Good one.
Sharp eye you have ;-)